ioread.cpp

```diff
@@ -1163,6 +1164,7 @@ bool exec_readGrid(dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geometry>& mpiGrid,
    if(success) { success=readCellParamsVariable(file,fileCells,localCellStartOffset,localCells,"max_v_dt",CellParams::MAXVDT,1,mpiGrid); }
    if(success) { success=readCellParamsVariable(file,fileCells,localCellStartOffset,localCells,"max_r_dt",CellParams::MAXRDT,1,mpiGrid); }
    if(success) { success=readCellParamsVariable(file,fileCells,localCellStartOffset,localCells,"max_fields_dt",CellParams::MAXFDT,1,mpiGrid); }
+   if(success) { success=readCellParamsVariable(file,fileCells,localCellStartOffset,localCells,"vg_alpha",CellParams::ALPHA,1,mpiGrid); }
```

Comment:
Hmm, with this line in place, the code won't ever be able to restart from old restart files that didn't write an alpha value.
Should there be a fallback behaviour if alpha is not found? Or would it be trivially possible to add a dummy alpha value to old restart files with a small python script, such that this is a non-problem?

Reply:
Fixed, now it just prints to cout if it doesn't find alpha. This leaves the variable initialized to zero for the cells.